perception/map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -911,15 +911,22 @@ PredictedObject MapBasedPredictionNode::convertToPredictedObject(
 {
   PredictedObject predicted_object;
   predicted_object.kinematics = convertToPredictedKinematics(tracked_object.kinematics);
-  predicted_object.classification = tracked_object.classification;
+  autoware_perception_msgs::msg::ObjectClassification classification;
+  for (size_t i = 0; i < tracked_object.classification.size(); i++) {
+    classification.label = tracked_object.classification[i].label;
+    classification.probability = tracked_object.classification[i].probability;
+    predicted_object.classification.push_back(classification);
+  }
```

Comment:
Note: Need to confirm why the writing style was changed.

Reply:
```
  predicted_object.classification = tracked_object.classification;
```
I have confirmed that it works correctly with the original code here with planning simulator.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/6893/commits/d258b4f0a560715b458d909a5968ab572d37ff09